app/templates/administrative-units/administrative-unit/core-data/edit.hbs

```diff
@@ -187,10 +187,9 @@
                   <Item @labelFor="purpose">
                     <:label>Doel</:label>
                     <:content>
-                      <RichTextEditor
-                        @onChange={{fn
-                          (mut @model.administrativeUnit.purpose)
-                        }}
+                      <AuTextarea
```

Comment:
I guess it's an error due to the merge conflict, but we do want to use the `RichTextEditor` component here